metadata_layer.py

```diff
@@ -159,9 +159,9 @@ def __init__(self, sort_name=None, display_name=None,
         self.lc = lc
         self.viaf = viaf
         self.biography = biography
-        self.aliases = aliases
+        self.aliases = aliases or []
```

Comment:
Attribute changes allow `ContributorData` to feign `Contributor` persona more readily.